bigchaindb/common/schema/transaction.yaml

```diff
@@ -105,13 +105,14 @@ definitions:
     description: |
         Description of the asset being transacted. In the case of a ``TRANSFER``
         transaction, this field contains only the ID of asset. In the case
-        of a ``CREATE`` transaction, this field may contain properties:
+        of a ``CREATE`` transaction, this field contains only the user-defined
+        payload and asset-related keywords.
     additionalProperties: false
-    required:
-    - id
```

Comment:
It is, but we don't currently have a good way to split the requirements based on the operation (see #946).